applications/PfemFluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -29,7 +29,8 @@
 #include "custom_processes/select_mesh_elements_for_fluids_process.hpp"
 #include "custom_processes/generate_new_nodes_before_meshing_process.hpp"
 #include "custom_processes/inlet_management_process.hpp"
-#include "custom_processes/set_inlet_process.hpp"
+#include "custom_processes/set_lagrangian_inlet_process.hpp"
+#include "custom_processes/set_eulerian_inlet_process.hpp"
```

Comment:
just for the record, it is usually advisable to implement .h and .cpp... Easier to maintan and faster to compile